spec/unit/token_spec.rb

```diff
@@ -17,4 +17,33 @@
       end
     end
   end
+
+  describe '.load' do
+    context 'token file exists' do
+      it 'reads the file, instantiates and returns a token' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.